tools/mason/MasonHelp.chpl

```diff
@@ -403,3 +405,43 @@ proc masonSystemPcHelp() {
 }
 
 
+proc masonPublishHelp(){
```

Comment:
Won't this conflict with https://github.com/chapel-lang/chapel/pull/13198 ?

I might leave the introduction of `mason publish` to that PR.